src/status_im/contexts/chat/messenger/messages/content/status_link_preview/view.cljs

```diff
@@ -0,0 +1,28 @@
+(ns status-im.contexts.chat.messenger.messages.content.status-link-preview.view
+  (:require [quo.core :as quo]
+            [react-native.core :as rn]
+            [utils.re-frame :as rf]))
+
+(defn view
+  [{:keys [chat-id message-id]}]
+  (let [status-link-previews (rf/sub [:chats/message-status-link-previews chat-id message-id])
+        link-previews?       (rf/sub [:chats/message-link-previews? chat-id message-id])]
+    (when (seq status-link-previews)
+      [rn/view {:style {:margin-top (when link-previews? 8)}}
```

Comment:
why `status-link-previews` margin depends on `link-previews`?

Reply:
Because we're rendering them on top of each other
And if there was a link preview existing we need to push the community ones down a bit.